### ISSUE TITLE
image: ^3.3.0 conflict

### ISSUE THREAD

**Issue by PROGrand** [chooyan-eng/crop_your_image#83]
image 3.3.0's exif update causes compilation errors in crop_your_image.

**Comment by JuanDavid1996SG** [comment]
I was trying to run my project with this library and I got the following error:

```
Running Gradle task 'assembleRelease'...                        
../../../.pub-cache/hosted/pub.dartlang.org/crop_your_image-0.7.3/lib/src/crop.dart:730:27: Error: The getter 'data' isn't defined for the class 'ExifData'.
 - 'ExifData' is from 'package:image/src/exif/exif_data.dart' ('../../../.pub-cache/hosted/pub.dartlang.org/image-3.3.0/lib/src/exif/exif_data.dart').
Try correcting the name to the name of an existing getter, or defining a getter or field named 'data'.
  switch (tempImage?.exif.data[0x0112] ?? -1) {
```

I don't use all the features of this library so I made a fork and commented out a piece of code.
https://github.com/JuanDavid1996SG/crop_your_image/commit/a87464b9b3635156ff1f25ee8672ad4afa948837

**Comment by fingerart** [comment]
Should be adjusted as follows:
```dart
switch (tempImage?.exif.imageIfd.data[0x0112] ?? -1)
```

**Comment by FabioLafayete** [comment]
For now, you can fix it by forcing the version

dependency_overrides:
  image: ^4.0.3

**Comment by rickypid** [comment]
hi @chooyan-eng,
i have the same issue.

**Comment by Linspirit98** [comment]
I have the same issue, any update?

**Comment by chooyan-eng** [comment]
Thank you all for the feedbacks. 
#84 was merged and I've released this fix as `0.7.4`.

**Comment by PROGrand** [comment]
epic updates, causing compilation issues on dependent packages should change first or second version chunk: better to choose 1.0.0 or 0.8.0 for crop_your_image new version.

**Comment by chooyan-eng** [comment]
@PROGrand 
Thank you for your advice. But unfortunately, I published the update as `0.7.4` before reading your reply. 
I'll be sure to change the first or second version next time. Thanks.
